wandb/integration/openai/openai.py

```diff
@@ -66,7 +67,9 @@ def create(*args, **kwargs):
             # save original method
             self.original_methods[symbol] = original
             # monkeypatch
-            getattr(self.openai, symbol).create = method_factory(original)
+            wrapper = method_factory(original)
+            wrapper = functools.update_wrapper(wrapper, original)
```

Comment:
curious why not just use `functools.wraps` on the create method? 
I guess this one will work for other functions we add to the factory out-of-the-box...

Reply:
I used this because it's easier to read and follow when compared to the wraps decorator. I will defer to your call here and use wraps instead.